clientapp/src/components/TracerForm.jsx

```diff
@@ -116,12 +121,16 @@ export class TracerForm extends React.Component {
     }
 
     onStartDateChange(ev) {
-        const newStartDate = ev.target.value;
+        var startDate = document.getElementById("start-date").value;
+        var startTime = document.getElementById("start-time").value;
+        const newStartDate = startDate + " " + startTime;
```

Comment:
There are two different ways we are using the date and time. One if for displaying it in the card and the other is for creating a Date object to store in the db, I think it's best that we keep these separate, since they may not necessarily be the same format. For example, right now we end up doing something like `new Date("07/28/2020 10:00)` which is an acceptable format, but if we want to change how it is displayed on the card, we don't want to also change how its passed to the constructor